### PR TITLE
[AutoParallel] Check llama cross entropy

### PR DESCRIPTION
**Pull request by LiYuRio**
### PR types

New features

### PR changes

Others

### Description

验证llama的子图cross entropy
Pcard-73145

### CODE REVIEW

File: test/auto_parallel/semi_auto_parallel_for_llama_mlp_with_cross_entropy_loss.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        super().__init__()
+        self.ignore_index = -100
+        self.loss_func = paddle.nn.CrossEntropyLoss(
+            reduction="none", ignore_index=self.ignore_index
+        )
+
+    def forward(self, prediction_scores, masked_lm_labels):
+        # TODO(liyurui): Unsqueeze inferspmd is not implemented yet. Uncomment this after we support.
+        # masked_lm_loss = self.loss_func(prediction_scores.astype("float32"), masked_lm_labels.unsqueeze(2))
+        masked_lm_loss = self.loss_func(
```
Comment (liuzhenhai93) [inline test/auto_parallel/semi_auto_parallel_for_llama_mlp_with_cross_entropy_loss.py:59]: 5651ac9c652366c98956e2fb18d814cb
Unsqueeze 实现了呀

Comment (LiYuRio) [reply test/auto_parallel/semi_auto_parallel_for_llama_mlp_with_cross_entropy_loss.py:59]: 下个pr修

**Review by liuzhenhai93 — APPROVED** [review]
LGTM

**Review by GhostScreaming — APPROVED** [review]
LGTM

**Pull request merged**